### PR TITLE
Add yarn workspaces

### PR DESCRIPTION
**Pull request by codinsonn**
Add yarn workspaces to monorepo setup (#6) for easy dependency management & script execution:
https://classic.yarnpkg.com/lang/en/docs/workspaces/

You can now start the client from the monorepo root with `yarn dev:client`

And instead of `cd {workspace} && yarn run {command}`, you can now do:
- `yarn meldet-client {command}` ➡️ e.g. `yarn meldet-client add run build`
- `yarn meldet-admin {command}` ➡️ e.g. `yarn meldet-client add -D types/react`
- `yarn meldet-api {command}` ➡️ e.g. `yarn meldet-client add prisma`

### CODE REVIEW

**Review by IljaCooreman — APPROVED** [review]
nice! PR is good for me, but wait for #6 to merge I guess?

**Comment by codinsonn** [comment]
> PR is good for me, but wait for #6 to merge I guess?

Can be merged into #6 and then be reviewed by @SimonRijsselaere & @fd  as 1 PR as well

**Comment by IljaCooreman** [comment]
@codinsonn merging into #6 is good for me! I'll let you merge it, since you're the owner of the PR

**Pull request merged**
